js/tsconfig/tsconfig.es2015.cls.json

```diff
@@ -2,10 +2,10 @@
 {
   "extends": "./tsconfig.base.json",
   "compilerOptions": {
-    "target": "ES2015",
```

Comment:
@trxcllnt why was this es2015? The output was always only used for closure so all `.cls.json` files should be identical, no?

Reply:
Apparently the documentation for this file was incorrect.